stuff.js

```diff
@@ -0,0 +1,36 @@
+// uploads image to AWS s3 and updates user schema with url of profile image
+router.post("/profileImageUpload", async function (req, res, next) {
+  if (req.files === null) {
```

Comment:
instead of `if (req.files === null) {` we could use `if (req.files == null) {` as it will check for both `undefined` and `null`

Reply:
I'm not a fan of using loose equality for anything. If we want to make the check more general we could just do `if (!req.files)`